tap-snapshots/tests/test_deployments.ts.test.cjs

```diff
@@ -58,7 +58,6 @@ Object {
                 },
               ],
               "signedBy": Object {
-                "allOf": Array [],
```

Comment:
Did you test a deployment with this?

Reply:
I think so. I did the same flow as we did together